routes/jwtauth/jwtgetUser.js

```diff
@@ -0,0 +1,53 @@
+import jwtVerify from "./jwtVerify";
+import { getClientIp } from "request-ip";
+import User from "../../models/user";
+import authLog from "../../models/authlog";
+import jwtVerifyAccess from "./jwtVerifyAccess";
+import moment from "moment";
+
+const jwtgetUser = async (req, jwtToken, options) => {
```

Comment:
Codacy found an issue: [Async arrow function has a complexity of 9. Maximum allowed is 4.](https://app.codacy.com/gh/implude/local-community-backend/pullRequest?prid=6225533)